File: Data Structures & Algorithms/squares-of-a-sorted-array/submission-0.java
class Solution {
    public int[] sortedSquares(int[] nums) {
        
        int []num = new int[nums.length];
        for( int i=0; i<nums.length; i++ ) {
             num[i] = Math.abs(nums[i]);
        }
        Arrays.sort(num);

         int []ans  = new int[num.length];
        for( int i=0; i<num.length; i++ ) {
            ans[i] = Math.abs(num[i]*num[i]);
        }

        return ans;
    }
}

File: Data Structures & Algorithms/valid-perfect-square/submission-0.cpp
class Solution {
public:
    bool isPerfectSquare(int num) {
        
        long long ans = 0;

        for( long long  i=1; i<=num; i++ ) {
            ans = i*i;
            if( ans == num  ) {
                return true;
            }
        }
        return false;
    }
};